SmartDeviceLink/SDLAudioStreamingIndicator.h

```diff
@@ -0,0 +1,36 @@
+//  SDLAudioStreamingIndicator.h
```

Comment:
The _SDLAudioStreamingIndicator.h_'s target membership needs to be changed to *public*.

Reply:
Yeah that header should be public. Done.